### DISCUSSION TITLE
Some questions (Shapes, 3D models with animations)

### DISCUSSION THREAD

**Discussion by Shibulski** [raysan5/raylib#1720] [Q&A] [answered]
- are the shapes implemented on GPU?
- whats the best pipeline to import 3D models with animation? I've my models right now in fbx format (which also include animations)

Thx in advance for the answers!

**Comment by raysan5** [comment, marked as answer]
@Shibulski answers:

 - Yes, everything that is drawn on the screen goes through the GPU.
 - raylib supports models with animations for the formats glTF and IQM, but not all features of those formats are supported.

**Reply by Shibulski** [reply]
Thanks!
